package.json

```diff
@@ -24,7 +24,7 @@
     "@verdaccio/local-storage": "8.4.2",
     "@verdaccio/readme": "8.4.2",
     "@verdaccio/streams": "8.2.0",
-    "@verdaccio/ui-theme": "0.3.2",
+    "@verdaccio/ui-theme": "0.3.9",
```

Comment:
The release could have been a minor one !!!! ...cause of that big feature (theme)

Reply:
Yeah agree, it should not, I will change the way we release ui-theme soon, it is not a minor but it how was published. My bad. Too late to rollback.

Reply:
Ohh okok. 👍